src/app/WriteHandler.h

```diff
@@ -35,6 +36,21 @@
 
 namespace chip {
 namespace app {
+
+class WriteHandler;
+
+class WriteHandlerDelegate
+{
+public:
+    virtual ~WriteHandlerDelegate() = default;
+
+    /**
+     * Returns whether the write operation to the given path is conflict with another write operations.
```

Comment:
```suggestion
     * Returns whether the write operation to the given path is in conflict with another write operation.
```
I know you just copied this...